ros2nodl/setup.py

```diff
@@ -0,0 +1,41 @@
+from setuptools import find_packages, setup
+
+package_name = 'ros2nodl'
+
+setup(
+    name=package_name,
+    version='0.0.1',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='Ubuntu Robotics',
+    maintainer_email='ubuntu-robotics@lists.launchpad.net',
+    url='https://github.com/ubuntu-robotics/nodl',
+    download_url='https://github.com/ubuntu-robotics/nodl/releases',
+    keywords=['ROS'],
+    classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU General Public License v3',
+        'Programming Language :: Python',
+        'Topic :: Software Development',
+    ],
+    description='CLI tools for NoDL files.',
+    license='GNU General Public License v3',
+    tests_require=['pytest'],
+    entry_points={
+        'console_scripts': [
+        ],
+        'ros2cli.command': [
+            'nodl = ros2nodl.command.nodl:NoDLCommand',
+        ],
+        'ros2nodl.verb': [
+            'show = ros2nodl.verb.show:ShowVerb',
+            'validate = ros2nodl.verb.validate:ValidateVerb'
```

Comment:
You have moved these to non-public, this probably won't work anymore. Make sure you test this.

Reply:
Fixed in 1b16352 , not sure if it was even causing any behavior though...